src/util-inl.h

```diff
@@ -336,7 +336,7 @@ inline size_t MultiplyWithOverflowCheck(size_t a, size_t b) {
 // nullptr for zero-sized allocation requests.  Normalize by always using
 // a nullptr.
 template <typename T>
-T* UncheckedRealloc(T* pointer, size_t n) {
+inline T* UncheckedRealloc(T* pointer, size_t n) {
```

Comment:
Does this change affect the warnings? This function doesn’t seem to be a particularly good candidate for inlining.